src/styles/_catalog.scss

```diff
@@ -1,22 +1,31 @@
-.catalog {
-  display: flex;
-  flex-wrap: wrap;
+$card-width: 200px;
 
-  max-width: 944px;
+.catalog {
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(4, $card-width);
+  gap: 48px;
   margin: 0 auto;
 
   &__card {
     display: flex;
     justify-content: center;
     align-items: center;
-    width: 200px;
+    width: $card-width;
```

Comment:
actually, just found it, you should not set width for your card anyways, it should be controlled by grid